pyworker/job.py

```diff
@@ -28,6 +28,8 @@ def __init__(self, class_name, database, logger,
         self.job_id = job_id
         self.job_name = '%s#run' % class_name
         self.attempts = attempts
+        if max_backoff_delay_seconds:
+            max_backoff_delay_seconds = max(max_backoff_delay_seconds, 2) # max_backoff_delay_seconds can not be less than 2 seconds
```

Comment:
- Can we make it 5 to match the internal calculation?
- Document this minimum in the README.
- Add a new test to ensure the minimum is honoured.